### PR TITLE
Update 29 casks to use versioned urls

### PR DESCRIPTION
**Pull request by victorpopkov**

### CODE REVIEW

File: Casks/aliwangwang.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask :v1 => 'aliwangwang' do
-  version :latest
-  sha256 :no_check
+  version '3.2.0-3714'
+  sha256 '66c85b4efbc2576d6c3a5a03b0b7a6ae1e6758d083c2547d854c489e00c03dc2'
 
-  # etao.com is the official download host per the vendor homepage
-  url 'http://update.labs.etao.com/macww-nosandbox/versions/lastest'
+  # alicdn.com is the official download host per the vendor homepage
+  url "https://dbison.alicdn.com/updates/macww-#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/aliwangwang.rb]: Can I get the download page where this `url` is from? I trust you, but it still needs to be verified by a maintainer.

File: Casks/automatic.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
 cask :v1 => 'automatic' do
-  version :latest
-  sha256 :no_check
+  version '2.4.6.391'
+  sha256 '4e3f82fb37276d8ae3f3a6ecdf7c30f0c3612c1696fe3f030b0804ecd0217a55'
 
-  url 'http://codingcurious.com/downloads/Automatic.2.zip'
+  # amazonaws.com is the official download host per the vendor homepage
+  url "https://com-codingcurious-downloads.s3.amazonaws.com/Automatic.#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/automatic.rb]: Also, the link for this one. The app appears to have changed name, so this cask should change as well.

File: Casks/bee.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 cask :v1 => 'bee' do
-  version :latest
-  sha256 :no_check
+  version '2.1'
+  sha256 '3a3103938959d212ac9eb7c78bf83931d2876384ee13f5ffd3a9b3b2f0da3d58'
 
-  url 'http://neat.io/bee/download.html'
+  url "http://bee-app.s3.amazonaws.com/public/Bee-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/bee.rb]: Missing the `# amazonaws.com is the official download host per the vendor homepage` comment.

File: Casks/bettertouchtool.rb [hunk 4]
```diff
@@ -5,9 +5,9 @@
     sha256 'fad5e9d36259c379bdb33188cf15d179fd9ff73023035c98f5734e7e3e13bb75'
     url "http://bettertouchtool.net/btt#{version}.zip"
   else
-    version :latest
-    sha256 :no_check
-    url 'http://bettertouchtool.net/BetterTouchTool.zip'
+    version '1.10'
+    sha256 '25f6fb864cb3dfb1711784be03a55f620bac4e43c06372cce8316380460f8ca1'
+    url "http://boastr.net/releases/btt#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/bettertouchtool.rb]: Missing the comment as well. Alternatively (and likely a better choice), `homepage` could be changed to `http://boastr.net/`, as could the previous `url`.

File: Casks/github.rb [hunk 5]
```diff
@@ -1,8 +1,9 @@
 cask :v1 => 'github' do
-  version :latest
-  sha256 :no_check
+  version '208'
+  sha256 '8c1f458d273d335a4983e5918374c9e0927e19ed5c86ca9607cd1aadc2d553da'
 
-  url 'https://central.github.com/mac/latest'
+  # amazonaws.com is the official download host per the vendor homepage
+  url "https://github-central.s3.amazonaws.com/mac/GitHub%20for%20Mac%20#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/github.rb]: Also need the page for this `url`, since I’m getting a `cloudfront` unversioned one.

File: Casks/aliwangwang.rb (hunk 1 above)
Comment (Ngrd) [inline Casks/aliwangwang.rb]: The original download url used in this cask (`update.labs.etao.com`) simply redirects to the `dbison.alicdn.com`.

File: Casks/automatic.rb (hunk 2 above)
Comment (Ngrd) [inline Casks/automatic.rb]: Yes, I did the research. The Otomatic's predecessor Automatic 2 is still available including the licenses with discounts (see "Looking for Automatic 2?" on their homepage). In this case, it is much better to make a separate cask for Otomatic.

File: Casks/github.rb (hunk 5 above)
Comment (Ngrd) [inline Casks/github.rb]: The original download url (host `central.github.com`) used in this cask simply redirects to the `github-central.s3.amazonaws.com`.

File: Casks/bee.rb (hunk 3 above)
Comment (Ngrd) [inline Casks/bee.rb]: Yes, thank you for noticing. Will definitely add it as soon as all other questions will be solved.

File: Casks/bettertouchtool.rb (hunk 4 above)
Comment (Ngrd) [inline Casks/bettertouchtool.rb]: Yes, thank you! Also thought about changing the homepage, but wasn't sure enough. Will definitely change it as soon as all other questions will be solved.

**Pull request merged**